Build/dex2jar/build.sh

```diff
@@ -35,7 +35,7 @@ Type=Application
 Terminal=false
 Icon[en_US]=terminator
 Name[en_US]=Dex2Jar
-Exec=/usr/bin/x-terminal-emulator --command "ls /usr/bin/d2j*; \$SHELL"
+Exec=x-terminal-emulator --command "cd /usr/bin; ls d2j*; $SHELL"
```

Comment:
1. We need that \ in front of $SHELL to escape $ in this script. 
2. We need a cd ~ before we do $SHELL to put the user back in his home directory. 
There is no point leaving user in /usr/bin/ it will just add more confusion.